<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hash_table</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ui_images/icon.jpg</normaloff>ui_images/icon.jpg</iconset>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>——————————————————————————————————————————————————————————</string>
   </property>
  </widget>
  <widget class="QPushButton" name="translate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>翻译一下→</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="english">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入英文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QListWidget" name="word_list">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chinese">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入中文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除单词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>——————————————————————————————————————————————————————————</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加单词</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delet_word">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入你要删除的单词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delet">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>单词查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_word">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入你要查询的单词</string>
   </property>
  </widget>
  <widget class="QLabel" name="word_num">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前已录入0个单词</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
